server/services/participants/participant-entries.ts

```diff
@@ -188,13 +186,17 @@ export const getParticipants = async (
   isIndigenousFilter?: IsIndigenousFilter,
   programFilter?: ProgramFilter
 ) => {
+  // MHAW program isn't open yet to other regions
+  const hcaOnly =
+    user?.isHA && !MHAW_ENABLED_REGIONS.some((region) => user?.roles.includes(region));
+
   // Get user ids
   const participantsFinder = new ParticipantsFinder(dbClient, user);
   const interestFilter =
     isPrivateEmployerOrMHSUEmployerOrHA(user) && statusFilters?.includes('open');
   let participants = await participantsFinder
     .filterRegion(regionFilter)
-    .filterProgram(programFilter)
+    .filterProgram(hcaOnly ? 'HCA' : programFilter)
```

Comment:
need to update the `isMHSUEmployer` check here to use `programFilter` if it exists, currently just hardcoded to `MHAW`